### ISSUE TITLE
Table not show Pagination if current not to be 1

### ISSUE THREAD

**Issue by shaodahong** [ant-design/ant-design#25836]
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link

When I changed Pagination current to 2（by fetch data），and in the second page only have one data, if I delete it and refresh table data, page display

![image](https://user-images.githubusercontent.com/14831261/88529683-eb216d00-d032-11ea-9e75-d03ce4ac1f18.png)

[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/pensive-feistel-4rmmg)

### Steps to reproduce
view table

### What is expected?
Pagination show

### What is actually happening?
Pagination not show

| Environment | Info |
|---|---|
| antd | 4.4.3 |
| React | 16.13.0 |
| System | Mac OS X |
| Browser | chrome 75.0.3770.100  |

**Comment by shaodahong** [comment]
In my see, show pagination logic maybe `{pageData && pageData.length  > 0 && pagination.current === 1 && bottomPaginationNode}`

**Comment by 07akioni** [comment]
`current = 1` doesn't work too. https://codesandbox.io/s/compassionate-silence-gt4yz?file=/index.js

**Comment by shaodahong** [comment]
> `current = 1` doesn't work too. https://codesandbox.io/s/compassionate-silence-gt4yz?file=/index.js

Because `pageData.length === 0`, but if `current === 2` and `pageData.length === 0`, I can't go back to the first page

**Comment by 07akioni** [comment]
why need you go back to first page when `pageData.length === 0`?

**Comment by shaodahong** [comment]
### A scene:

Pagination shown when `pageData.length !== 0`

`current === 2` and `pageData.length === 1`(get data by some http fetch, like `http.get({current: 2}) => [data]`)

when i delete data(by `http.delete()`) and refresh table(by `http.get({current: 2}) => []` ), Pagination won't show

**Comment by 07akioni** [comment]
https://codesandbox.io/s/recursing-swanson-4l3kg?file=/index.js

**Comment by sirlantis** [comment]
It would be nice if the Table pagination would [follow the same logic as `RcPagination` itself](https://github.com/react-component/pagination/blob/42d979fd4661b8720b36565d61cbb0cb6d573ba1/src/Pagination.jsx#L379) (and as documented in Antd): 

```
    // When hideOnSinglePage is true and there is only 1 page, hide the pager
    if (hideOnSinglePage === true && total <= pageSize) {
      return null;
    }
```

**Comment by zhangchen915** [comment]
Just remove pageData.length like topPaginationNode?
https://github.com/ant-design/ant-design/blob/1119c3da385f6d6711f1d392a930b6053bb9c6a8/components/table/Table.tsx#L498

https://github.com/ant-design/ant-design/blob/1119c3da385f6d6711f1d392a930b6053bb9c6a8/components/table/Table.tsx#L476

**Comment by shaodahong** [comment]
> Just remove pageData.length like topPaginationNode?
> https://github.com/ant-design/ant-design/blob/1119c3da385f6d6711f1d392a930b6053bb9c6a8/components/table/Table.tsx#L498
> 
> https://github.com/ant-design/ant-design/blob/1119c3da385f6d6711f1d392a930b6053bb9c6a8/components/table/Table.tsx#L476

Yep, Send PR Welcome😁

**Comment by sirlantis** [comment]
This issue is not fixed - #26126 only works for locally filtered and paginated data.

**Comment by sirlantis** [comment]
> > Just remove pageData.length like topPaginationNode?
>
> Yep, Send PR Welcome

I can put up a PR for removing this, but it would directly contradict the change made here: https://github.com/ant-design/ant-design/pull/22548
